src/molecules/todoList/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Components
+import TodoItem from '../todoItem';
+
+// Styles
+import "./TodoList.css"
+
+const TodoList = ({todoList, onTodoDelete}) => {
```

Comment:
In general any component which accepts props should mention their type(s) and default value(s). We can achieve this by importing PropTypes from package names prop-types. So TodoList component can actually have these before its export.

// Rest Definition of Component

TodoList.propTypes = {
    todoList: PropTypes.array
}

TodoList.defaultProps = {
    todoList: EMPTY_ARRAY
}

export default TodoList;